doc/source/tutorial/optimize.rst

```diff
@@ -1106,6 +1106,183 @@ For example, to find the minimum of :math:`J_{1}\left( x \right)` near
     5.33144184241
 
 
+Optimization Solvers Reference Tables
+-------------------------------------
+
+Provide an overview of various optimization solvers,which are tools 
+used to find the minimum or maximum of an objective function subject 
+to constraints. These solvers are divided into two main categories: 
+Local Optimization and Global Optimization.Each category has a set of 
+solvers with unique characteristics and capabilities.
+
+
+Local Optimization Solvers
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Local optimization solvers are algorithms that search for a local minimum, 
+starting from an initial point. These solvers are particularly useful 
+when the problem is presumed to have a smooth landscape around the initial guess.
+
+**Local Optimization Solvers Table:**
+
+.. list-table::
+   :widths: 15 15 15 20 20 15
+   :header-rows: 1
+
+   * - Solver
+     - Uses Gradient
+     - Uses Hessian
+     - Bounds Constraints
+     - Nonlinear Constraints
+     - Utilizes Sparsity
+   * - Nelder-Mead
+     - 
+     - 
+     - ✓
+     - 
+     - 
+   * - Powell
+     - 
+     - 
+     - ✓
+     - 
+     - 
+   * - COBYLA
+     - 
+     - 
+     - ✓
+     - 
+     - 
+   * - CG
+     - ✓
+     - 
+     - 
+     - 
+     - 
+   * - BFGS
+     - ✓
+     - 
+     - 
+     - 
+     - 
+   * - Newton-CG
+     - ✓
+     - ✓
+     - 
+     - 
+     - ✓
+   * - dogleg
+     - ✓
+     - must be pos. def.
+     - 
+     - 
+     - 
+   * - trust-ncg
+     - ✓
+     - ✓
+     - 
+     - 
+     - 
+   * - trust-krylov
+     - ✓
+     - ✓
+     - 
+     - 
+     - 
+   * - trust-exact
+     - ✓
+     - ✓
+     - 
+     - 
+     - 
+   * - L-BFGS-B
+     - ✓
+     - 
+     - ✓
+     - 
+     - 
+   * - TNC
+     - ✓
+     - 
+     - ✓
+     - 
+     - 
+   * - trust-constr
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+   * - SLSQP
+     - ✓
+     - 
+     - ✓
+     - ✓
+     - 
+
+
+Global Optimization Solvers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Global optimization solvers aim to find the global minimum over the entire search space,
+which is especially important when the objective function has multiple local minima.
+
+**Global Optimization Solvers Table:**
+
+.. list-table::
+   :widths: 20 15 15 20 20 15
+   :header-rows: 1
+
+   * - Solver
+     - Uses Gradient
+     - Uses Hessian
+     - Bounds Constraints
+     - Nonlinear Constraints
+     - Utilizes Sparsity
+   * - dual_annealing
+     - (✓)
+     - (✓)
+     - ✓
+     - 
+     - 
+   * - differential_evolution
+     - 
+     - 
+     - ✓
+     - ✓
+     - 
+   * - shgo
+     - (✓)
+     - (✓)
+     - ✓
+     - ✓
+     - 
+   * - direct
+     - 
+     - 
+     - ✓
+     - 
+     - 
+   * - basinhopping
+     - (✓)
+     - (✓)
+     - 
+     - 
+     - 
+(✓): Depending on the chosen local minimizer
```

Comment:
I guess you have a minor doc build issue in CI: `/home/circleci/repo/doc/source/tutorial/optimize.rst:1272: WARNING: Explicit markup ends without a blank line; unexpected unindent.`

There are a few places in the first paragraph above where spaces are missing around punctuation, but that's pretty minor too.

Reply:
Thank you for your review. Fixed.